Orange/widgets/webview.py

```diff
@@ -1,91 +1,3 @@
-        The QObject to use as a parent. This object is also exposed
-        as ``window.pybridge`` in JavaScript.
-    html: str
-        The HTML to load the view with.
-    debug: bool
-        Whether to enable inspector tools on right click.
-    **kwargs:
-        Passed to QWebView.
-    """
-    def __init__(self, parent=None, bridge=None, html='', *, debug=False, **kwargs):
-        super().__init__(parent,
-                         sizePolicy=QSizePolicy(QSizePolicy.Expanding,
-                                                QSizePolicy.Expanding),
-                         sizeHint=QSize(500, 400),
-                         contextMenuPolicy=Qt.DefaultContextMenu,
-                         **kwargs)
-
-        if isinstance(parent, QWidget) and parent.layout() is not None:
-            parent.layout().addWidget(self)
-
-        self.bridge = bridge
-        self.frame = None
-
-        def _onload(_ok):
-            if not _ok: return
-            self.frame = self.page().mainFrame()
-            self.frame.javaScriptWindowObjectCleared.connect(
-                lambda: self.frame.addToJavaScriptWindowObject('pybridge', bridge))
-
-        self.loadFinished.connect(_onload)
-        _onload(True)
-
-        history = self.history()
-        history.setMaximumItemCount(0)
-        settings = self.settings()
-        settings.setMaximumPagesInCache(0)
-        settings.setAttribute(settings.LocalContentCanAccessFileUrls, True)
-        settings.setAttribute(settings.LocalContentCanAccessRemoteUrls, False)
-
-        if debug:
-            settings.setAttribute(settings.LocalStorageEnabled, True)
-            settings.setAttribute(settings.DeveloperExtrasEnabled, True)
-            settings.setObjectCacheCapacities(4e6, 4e6, 4e6)
-            settings.enablePersistentStorage()
-
-        if html:
-            self.setHtml(html)
-
-    def setContent(self, data, mimetype, base_url=''):
-        """Set the content `data` of type `mimetype` in the current webframe."""
-        super().setContent(data, mimetype, QUrl(base_url))
-
-    def dropEvent(self, event):
-        """Prevent loading of drag-and-drop dropped file"""
-        pass
-
-    def evalJS(self, code):
-        """Evaluate JavaScript code `code` in the current webframe and
-        return the result of the last executed statement."""
-        return self.frame.evaluateJavaScript(code)
-
-    def setHtml(self, html, base_url=''):
-        """Set the HTML content of the current webframe to `html`
-        (an UTF-8 string)."""
-        self.setContent(html.encode('utf-8'), 'text/html', base_url)
-
-    def svg(self):
-        """ Return SVG string of the first SVG element on the page, or
-        raise ValueError if not any. """
-        html = self.frame.toHtml()
-        return html[html.index('<svg '):html.index('</svg>') + 5]
+# This was once here. For compatibility:
+# pylint: disable=unused-import
+from Orange.widgets.utils.webview import WebviewWidget as WebView
```

Comment:
Gah! It was meant to, except in the two constructor arguments bit. [Word Cloud had been prepared for it](https://github.com/biolab/orange3-text/pull/99/files), but I forgot about Geo Map (fix in https://github.com/biolab/orange3-text/pull/115). Thanks for checking. Have now removed this file altogether.